### ISSUE TITLE
path.join problems compile on windows

### ISSUE THREAD

**Issue by pdietsch** [leafymind/gulp-sass-themes#3]
Hi Benjamin

On Windows the path join always produce a backslash @import "themes\red". I change the code:

```
themeImports[themeName] = new Buffer(`$current-theme-name: "${themeName}"; @import "${slash(path.join(themesPath, themeName))}"; 

function slash(str) {
    var isExtendedLengthPath = /^\\\\\?\\/.test(str);
    var hasNonAscii = /[^\x00-\x80]+/.test(str);

    if (isExtendedLengthPath || hasNonAscii) {
        return str;
    }

    return str.replace(/\\/g, '/');
};
```

([https://github.com/sindresorhus/slash/blob/master/index.js])

**Comment by bbuhler** [comment]
Hi @pdietsch,

thanks for reporting.
I will make use of sindresorhus' slash module/function as you suggested.

**Comment by bbuhler** [comment]
should be fixed with v1.2.1
